spec/controllers/verify/review_controller_spec.rb

```diff
@@ -169,77 +178,31 @@ def show
     context 'user has completed all steps' do
       before do
         idv_session.params = user_attrs
+        idv_session.applicant = idv_session.applicant_from_params
         stub_analytics
         allow(@analytics).to receive(:track_event)
       end
 
       it 'redirects to questions path' do
         put :create, user: { password: ControllerHelper::VALID_PASSWORD }
 
-        result = {
-          success: true,
-          idv_attempts_exceeded: false
-        }
-
-        expect(@analytics).to have_received(:track_event).with(Analytics::IDV_INITIAL, result)
+        expect(@analytics).to have_received(:track_event).with(Analytics::IDV_REVIEW_COMPLETE)
         expect(response).to redirect_to verify_questions_path
       end
     end
 
-    context 'user attributes fail to resolve' do
-      before do
-        idv_session.params = user_attrs.merge(first_name: 'Bad')
-        stub_analytics
-        allow(@analytics).to receive(:track_event)
-      end
-
-      it 'redirects to retry' do
-        put :create, user: { password: ControllerHelper::VALID_PASSWORD }
-
-        result = {
-          success: false,
-          idv_attempts_exceeded: false
-        }
-
-        expect(@analytics).to have_received(:track_event).with(Analytics::IDV_INITIAL, result)
-        expect(response).to redirect_to verify_retry_url
-      end
-
-      context 'max attempts exceeded' do
-        before do
-          user.idv_attempts = 3
-        end
-
-        it 'redirects to fail' do
-          put :create, user: { password: ControllerHelper::VALID_PASSWORD }
-
-          result = {
-            success: false,
-            idv_attempts_exceeded: true
-          }
```

Comment:
I found it, but it's expecting that analytics are not called, which doesn't sound right to me.